src/stores/DetailsUsage/shared.ts

```diff
@@ -0,0 +1,27 @@
+import { DataGrains } from 'components/graphs/types';
+import { PersistOptions } from 'zustand/middleware';
+import { DetailsUsageState } from './types';
+
+// Previous persist states for testing migrations
+// v0 - {"state":{"range":48,"statType":"docs","foo":"sup"},"version":0}
+export const persistOptions: PersistOptions<DetailsUsageState> = {
+    name: 'estuary.details-usage-store',
+    version: 1,
+    migrate: (persistedState: any, persistedVersion) => {
+        if (persistedVersion === 0 && persistedState) {
+            const amount = persistedState.range ?? 6;
+
+            // Version 0 was only storing the range as 6, 12, 24, 48
+            persistedState.range = {
+                amount,
+                grain: DataGrains.hourly,
+            };
+
+            if (persistedState.foo) {
+                delete persistedState.foo;
+            }
```

Comment:
I keep forgetting I left this in.... :facepalm:

Reply:
Haha are you taking it out?

Reply:
Nope... mainly talking about the `foo` prop that made it into the store and made it into prod local storage for a little while.